webpack.config.js

```diff
@@ -22,6 +23,25 @@ const webpackAssets =  {
   stylesheets: []
 }
 
+// The modules listed below should only take the smae source.
+// The alias would be like:
+// {
+//   'react': path.resolve(__dirname, 'node_modules', 'react'),
+//   'react-router': path.resolve(__dirname, 'node_modules', 'react-router')
+// }
+const singleSourceModuleAliasForDev = reduce([
+  'react',
+  'redux',
+  'react-redux',
+  'styled-components',
+  'react-router-dom',
+  'react-router',
+  'axios'
+], (prevAlias, moduleName) => ({
+  ...prevAlias,
+  [moduleName]: path.resolve(__dirname, 'node_modules', moduleName)
+}), {})
```

Comment:
I found that this works for Webpack to build bundle. But we still may encounter some errors when server side rendering. 

```bash
make start-dev-server
 [+] start dev server by nodemon src/server.js

NODE_ENV=development RELEASE_BRANCH= RENDER_ENV=server node_modules/.bin/nodemon --delay 1800ms src/server.js --exec node_modules/.bin/babel-node
[nodemon] 1.18.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node_modules/.bin/babel-node src/server.js`
module.js:549
    throw err;
    ^

Error: Cannot find module 'redux'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/mac2016cjyu/Repos/twreporter-npm-packages/packages/redux/lib/store/create-store.js:8:14)
    at Module._compile (module.js:652:30)
    at loader (/Users/mac2016cjyu/Repos/twreporter-react/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/mac2016cjyu/Repos/twreporter-react/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
[nodemon] app crashed - waiting for file changes before starting...
```
I'm trying to fix this.